### ISSUE TITLE
Create one index per site

### ISSUE THREAD

**Issue by dfeyer** [Flowpack/Flowpack.ElasticSearch.ContentRepositoryAdaptor#177]
Currently we use only one index for all sites, that can make some scoring feature of Elastic to not work as expected, by ex. word frequency will take in account other site, ...

**Comment by kdambekalns** [comment]
This refers to multi-site setups, i.e. where more than one site is in the same instance of Neos, right?

**Comment by daniellienert** [comment]
Yes, that's the use case.

**Comment by dfeyer** [comment]
Thinking about this issue, because I'm in the "one index per dimensions combination" ... it can be done, but we will couple this package with Neos ... a site is nothing for the ContentRepository :(

**Comment by kdambekalns** [comment]
True, but… we could still allow to define "scopes" for "sub-trees", no?

**Comment by dfeyer** [comment]
I restart working on the "one index per dimensions preset", and I will try to experiment a bit in implementing a `IndexNameSelectorInterface` that will make this possible ... but first let's make the current PR stable before breaking everything again ;)
